src/main/resources/init_db.sql

```diff
@@ -0,0 +1,5 @@
+CREATE SCHEMA IF NOT EXISTS test;
+USE test;
+CREATE TABLE IF NOT EXISTS book (id BIGINT AUTO_INCREMENT PRIMARY KEY,
+                                  title VARCHAR(255) NOT NULL,
+                                  price DECIMAL(10, 2) NOT NULL);
```

Comment:
it is better to change your formatting
```suggestion
CREATE TABLE IF NOT EXISTS book (
  id BIGINT AUTO_INCREMENT PRIMARY KEY,
  title VARCHAR(255) NOT NULL,
  price DECIMAL(10, 2) NOT NULL
);
```